src/sc/embedder/adapters.py

```diff
@@ -3,28 +3,25 @@
 from lxml import etree
 from lxml import html
 from Products.TinyMCE.adapters.interfaces.JSONDetails import IJSONDetails
+from Products.TinyMCE.adapters.JSONDetails import JSONDetails
```

Comment:
could you please rename this as `JSONDetailsBase`?

Reply:
Done